Tetrahedral_remeshing/examples/Tetrahedral_remeshing/tetrahedral_remeshing_of_one_subdomain.cpp

```diff
@@ -41,17 +34,53 @@ struct Cells_of_subdomain_pmap
   }
 };
 
+Subdomain_index subdomain_on_side_of_plane(const K::Point_3& p,
+                                           const K::Plane_3& plane)
+{
+  if (plane.has_on_positive_side(p))
+    return 1;
+  else
+    return 2;
+}
+
+K::Point_3 centroid(const Remeshing_triangulation::Cell_handle c)
+{
+  return CGAL::centroid(c->vertex(0)->point(),
+                        c->vertex(1)->point(),
+                        c->vertex(2)->point(),
+                        c->vertex(3)->point());
+}
+
 int main(int argc, char* argv[])
 {
-  const double target_edge_length = (argc > 1) ? atof(argv[1]) : 0.1;
+  const double target_edge_length = (argc > 1) ? atof(argv[1]) : 0.05;
   const std::size_t nbv = (argc > 2) ? atoi(argv[2]) : 1000;
 
+  /// Create a randomly generated triangulation of a sphere
   Remeshing_triangulation tr;
-  CGAL::Tetrahedral_remeshing::generate_input_two_subdomains(nbv, tr);
+  CGAL::Tetrahedral_remeshing::insert_random_points_in_cube(nbv, tr);
 
-  CGAL::tetrahedral_isotropic_remeshing(tr, target_edge_length,
+  /// Use vertical plane to split the mesh into two subdomains
+  const K::Plane_3 plane(0, 0, 1, 0);
+  CGAL::Tetrahedral_remeshing::insert_points_on_plane(plane, nbv/10, tr);
```

Comment:
C:\CGAL_ROOT\CGAL-6.0-Ic-93\test\Tetrahedral_remeshing_Examples\tetrahedral_remeshing_of_one_subdomain.cpp(65,65): warning C4244: 'argument': conversion from 'double' to 'const size_t', possible loss of data [C:\CGAL_ROOT\CGAL-6.0-Ic-93\cmake\platforms\MSVC-2022-Preview-Release\test\Tetrahedral_remeshing_Examples\tetrahedral_remeshing_of_one_subdomain.vcxproj]